src/ape/api/providers.py

```diff
@@ -180,7 +180,7 @@ def get_balance(self, address: str) -> int:
         """
 
     @abstractmethod
-    def get_code(self, address: str) -> bytes:
+    def get_code(self, address: AddressType) -> bytes:
```

Comment:
```suggestion
    def get_code(self, address: AddressType) -> ContractCode:
```

Reply:
I think this should be resolved in newest commit.